xknx/io/disconnect.py

```diff
@@ -7,19 +7,23 @@
 class Disconnect(RequestResponse):
     """Class to send a DisconnectRequest and wait for a DisconnectResponse."""
 
-    def __init__(self, xknx, udp_client, communication_channel_id):
+    def __init__(self, xknx, udp_client, communication_channel_id, net_bind=None):
         """Initialize Disconnect class."""
         self.xknx = xknx
         self.udp_client = udp_client
-        super().__init__(xknx, self.udp_client, DisconnectResponse)
+        super().__init__(xknx, self.udp_client, DisconnectResponse, net_bind=net_bind)
         self.communication_channel_id = communication_channel_id
 
     def create_knxipframe(self) -> KNXIPFrame:
         """Create KNX/IP Frame object to be sent to device."""
         (local_addr, local_port) = self.udpclient.getsockname()
+        if self.net_bind:
+            endpoint = HPAI(ip_addr=self.net_bind[0], port=self.net_bind[1])
+        else:
+            endpoint = HPAI(ip_addr=local_addr, port=local_port)
```

Comment:
```suggestion
        if self.net_bind:
            (local_addr, local_port) = net_bind
        else:
            (local_addr, local_port) = self.udpclient.getsockname()
        endpoint = HPAI(ip_addr=local_addr, port=local_port)
```
we can omit the call to udpclient.getsockname() if we don't use the results anyway. Maybe change the variable names 'local_*' then.